app/helpers/dropdown/base.rb

```diff
@@ -98,18 +98,27 @@ def sub_items?
 
     def render(template)
       template.content_tag(:li, class: css_class) do
-        template.safe_join([link(template, label, url, options),
+        template.safe_join([link(template),
                             render_sub_items(template)].compact)
       end
     end
 
-    def link(template, label, url, options)
+    def link(template)
```

Comment:
Find ich etwas scary hier die Methodendefinition zu ändern da das hier ja eine Base Klasse ist von der wahrscheinlich auch in ein paar Wagons geerbt wird.. Hast du da drauf geschaut und durch gegrept?
An sich wär es ja auch kein Problem das vorerst so wie vorher zu behalten oder?